### DISCUSSION TITLE
Add support for Google Translation and Deepl

### DISCUSSION THREAD

**Discussion by dgillier** [MohmmedAshraf/laravel-translations#29] [Ideas]
Will be great to have a way to translate a key using Google Translate and/or Deepl with config provided keys...

Thanks for considering :)
Denis

**Comment by MohmmedAshraf** [comment]
Hey @dgillier I've already some plans for the package including the feature you mentioned stay tuned for updates

**Comment by MohmmedAshraf** [comment]
Hey @dgillier this has been implemented on #50
